### PR TITLE
ffmpeg: optimize cuda filters

### PR DESCRIPTION
**Pull request by Andarwinux**

### CODE REVIEW

**Comment by shinchiro** [comment]
This looks unnecessary. If there's performance it might be negligible

**Comment by Andarwinux** [comment]
There does seem to be some performance gain for yadif_cuda, though it seems to be hard to measure as the NVIDIA driver seems have driver level PGO that gives a little performance gain for running unoptimized ffmpeg after running optimized ffmpeg benchmarks?

File: packages/ffmpeg.cmake [hunk 1]
```diff
@@ -110,8 +110,10 @@ ExternalProject_Add(ffmpeg
         --disable-vaapi
         --disable-vdpau
         --disable-videotoolbox
+        --disable-ptx-compression
         --disable-decoder=libaom_av1
         --extra-cflags='-Wno-error=int-conversion'
+        --nvccflags='-O3'
```
Comment (NotTsunami) [inline packages/ffmpeg.cmake:116]: I've read that O3 is actually the default optimization level for nvcc...are you sure this is doing anything?

Comment (Andarwinux) [reply packages/ffmpeg.cmake:116]: https://github.com/FFmpeg/FFmpeg/blob/master/configure#L4528

**Pull request closed without merging**